### PR TITLE
ml stack release blog

### PR DESCRIPTION
**Pull request by ChuckHend**

### CODE REVIEW

File: docusaurus/blog/2024-01-15-clickbait-classifier/2024-01-15-clickbait-classifier.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+);
+ predict 
+---------
+       1
+(1 row)
+```
+
+There we go, a click bait classifier in Postgres!
+
+## Serve the model w/ a REST api using PostgREST
```
Comment (samay-sharma) [inline docusaurus/blog/2024-01-15-clickbait-classifier/2024-01-15-clickbait-classifier.md:379]: I'd like to understand why this is important? Is this part of a typical ML workflow for ML engineers?

**Review by samay-sharma — COMMENTED** [review]
Did one pass of review on it. Mainly about shortening it to convey key points and putting the rest as scripts in an examples folder or something which we link to.

That way it remains reproducible but isn't something everybody has to read through to get the point.

Will do copyediting pass over it once we apply this round of feedback.

File: docusaurus/blog/2024-01-15-clickbait-classifier/2024-01-15-clickbait-classifier.md (hunk 1 above)
Comment (ChuckHend) [reply docusaurus/blog/2024-01-15-clickbait-classifier/2024-01-15-clickbait-classifier.md:379]: Building a rest api over a ML model is very common task for ML engineer. REST API is what is in our embedding app, its how you reach OpenAI, hugging face, etc. Its how most teams share ML model between MLE team and backend eng. Alternative to how we are setting this up would be to share DB credentials, set up role access, have consumers of the ML model figure use postgres driver, handle calling a sql from their code...this is way simpler, lower complexity from user perspective.

https://ubuntu.com/blog/guide-to-ml-model-serving
https://www.iguazio.com/glossary/model-serving/

File: docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+---
+slug: text-classification
+title: 'Simplify your ML workflow with Postgres'
+authors: [adam]
+tags: [postgres, machine-learning, text-classification]
+---
+
+At [Tembo](https://tembo.io), we're building the platform that gives developers a Postgres experience that is optimized for their application. And today's applications are growing in complexity, especially with the rise of LLMs. For those who have been working in the ML space for a while, you know that the ML lifecycle is a complex process. Thats why we're releasing the [Tembo ML Stack](https://cloud.tembo.io), which gives you Postgres that is equipped with the tools to make ML lifecycle efficient and easy, all from Postgres.
```
Comment (samay-sharma) [inline docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md:8]: Can we have a arch diagram which shows that it's a complex process?

File: docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+## Frictionless ML Development with Postgres
+
+ML workflows are generally split into three phases; exploration, model training, model serving. Exploring data within Postgres is a familiar process to many scientists, developers and engineers. But training and serving models is often done outside of Postgres, requiring additional infrastructure and engineering work. Serving is typically done outside Postgres as well, whether its hosting a model HTTP server or the dreaded 'rewrite' of the model training from Python into another language. This often results in the people doing exploration and model training being different from the people who are responsible for serving the model, which leads to a lot of friction and inefficiency in the ML workflow.
```
Comment (samay-sharma) [inline docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md:12]: Could you bring in some example tools people would use for training and serving?

File: docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+The Tembo ML Stack brings the entire ML workflow into Postgres, making it easier to build and deploy ML models.  It puts operations and model serving in the hands of the model developer.
```
Comment (samay-sharma) [inline docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md:14]: Let's also say it includes pl/python allowing you to write python functions to do data transformations within Postgres.

File: docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md (hunk 2 above)
Comment (samay-sharma) [inline docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md:8]: Question, not suggestion:

For non-ML experts, is there value in writing that this covers end-to-end ML ops and not just searching of embeddings from a pre-trained model. Please look at our Vector stack docs if you want to dive into the latter.

File: docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md (hunk 3 above)
Comment (samay-sharma) [reply docusaurus/blog/2024-01-17-simplified-ml-workflow/index.md:12]: I know you mention them below but just hammering the point here might be a good idea.

**Review by samay-sharma — COMMENTED** [review]
Mostly looks good. I have a bunch of copy-editing feedback which is hard to give in github. Can you create a google docs version addressing the points I mentioned and I can quickly light it up with suggestions?

PS: I've been impressed by @theory 's copy editing feedback on my blog. He's definitely made it shorter and more concise. Maybe worth him having a look as well :)

File: docusaurus/blog/2024-01-18-simplified-ml-ops/index.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+An ML platform consists of all the tools and systems to manage the lifecycle of data science and machine learning projects. That lifecycle is a continuous and iterative process, from defining a business problem through ML model development, deploying for inference, and continuous monitoring and analysis. This incredibly complex process can become a barrier to entry for many teams. That’s why we built the [Tembo ML Stack](https://tembo.io/docs/tembo-stacks/machine-learning), which offers a Postgres configuration equipped with the tools to make ML lifecycle efficient and easy, all from Postgres. You can try it out today by signing up for [Tembo Cloud](cloud.tembo.io).
```
[the remark this answers is not in the document]
Comment (ChuckHend) [reply docusaurus/blog/2024-01-18-simplified-ml-ops/index.md:10]: think its fixed now?

**Review by samay-sharma — APPROVED** [review]

**Pull request merged**